src/ext/yuni/src/yuni/cmake/core/check-includes.cmake

```diff
@@ -47,6 +47,9 @@ check_include_file_cxx(iostream YUNI_HAS_IOSTREAM)
 # cassert
 check_include_file_cxx(cassert YUNI_HAS_CASSERT)
 
+# cassert
+check_include_file_cxx(string YUNI_HAS_STD_STRING)
+
```

Comment:
One can always assume that header <string> exists, please remove this.

Reply:
Indeed, I removed this unnecessary complexity.